Dockerfile

```diff
@@ -61,8 +58,12 @@ ENV PATH /edx/app/edxapp/edx-platform/bin:${PATH}
 ENV SETTINGS production
 RUN mkdir -p /edx/etc/
 
+ENV VIRTUAL_ENV=/venv
```

Comment:
Could we switch this to create the virtualenv in `/edx/app/edxapp/venvs/edxapp` instead to match our other installation methods?  This came up recently in a couple of other PR reviews.

Reply:
sure